### DISCUSSION TITLE
云服务器上启动milvus遇到错误，

### DISCUSSION THREAD

**Discussion by MockerPeking** [milvus-io/milvus#17994] [Q&A and General discussion]
在百度ls云服第一次可以安装并启动。试用attu遇到错误，与下文信息类似，后重启服务器，再次启动milvus  报的似乎是许可错误，这是？这种服务器不支持？

:~#sudo docker-compose up -d
Starting milvus-minio ... error
Starting milvus-etcd  ... 

Starting milvus-etcd  ... error

ERROR: for milvus-etcd  Cannot start service etcd: failed to start shim: start failed: : fork/exec /usr/bin/containerd-shim-runc-v2: permission denied: unknown

ERROR: for minio  Cannot start service minio: failed to start shim: start failed: : fork/exec /usr/bin/containerd-shim-runc-v2: permission denied: unknown

ERROR: for etcd  Cannot start service etcd: failed to start shim: start failed: : fork/exec /usr/bin/containerd-shim-runc-v2: permission denied: unknown
ERROR: Encountered errors while bringing up the project.

**Comment by MockerPeking** [comment]
docker   
 Server Version: 20.10.17
 runc version: v1.1.2-0-ga916309
 init version: de40ad0
 Profile: default
 Kernel Version: 5.4.0-109-generic
 Operating System: Ubuntu 20.04 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 2

**Comment by xiaofan-luan** [comment]
ERROR: for etcd Cannot start service etcd: failed to start shim: start failed: : fork/exec /usr/bin/containerd-shim-runc-v2: permission denied: unknown
seems to be a permission issue

**Reply by MockerPeking** [reply]
basically,  its hard encounter  "permission denied" error, my  cloud machine  seems  hard to encounter it either,  but it's a   Average performance cloud server  which  has 2 cpu cors and 4G ram,   does it exceed the limitation？
